### PR TITLE
Adding expand icon for RHS

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys, can we get a spinmint up for this? My spinmint is not working for some reason. No need to merge this yet.

### CODE REVIEW

**Comment by asaadmahmoodspin** [comment]
Guys, we can merge this in now. It's pretty much okay, we can create another ticket for someone to add the channel switching thing.

**Comment by it33** [comment]
Tool tip looks like an error, propose limiting the width? 

image

**Comment by it33** [comment]
In my mind, the expanded RHS and center channel look too similar. 

Wondering your thoughts on maybe leaving a gap? 

image

**Comment by it33** [comment]
btw, this is really nice work, search results are so much nicer now, 
 
image

**Comment by it33** [comment]
Actually for the LHS tab fix, wondering if we could do that as a branch off of this one, rather than merging something to master before it's passed the quality bar?

**Comment by it33** [comment]
K, more testing: 

#### 1) For the animation when the expand happens, instead of the grey area moving out from RIGHT to LEFT, perhaps we could try fading in without movement? 

On a large screen it looks fine, but when you shrink the screen to the size below it looks weird--maybe check and see if you agree? 

image

#### 2) Instead of "Shrink Sidebar" perhaps "Collapse Sidebar"?

**Comment by it33** [comment]
+1 

Beautiful work. Tested on Chrome and IE11 on Win7, everything works.

**Comment by lfbrock** [comment]
+1, tested on Safari.

**Comment by enahum** [comment]
+1

File: webapp/components/get_link_modal.jsx [hunk 1]
```diff
@@ -28,6 +29,13 @@ export default class GetLinkModal extends React.Component {
         this.props.onHide();
     }
 
+    copyLinkOnClick() {
```
Comment (crspeller) [inline webapp/components/get_link_modal.jsx]: This seems unrelated.

File: webapp/components/change_url_modal.jsx [hunk 2]
```diff
@@ -154,6 +155,7 @@ export default class ChangeUrlModal extends React.Component {
                             <div className='col-sm-10'>
                                 <div className={urlClass}>
                                     <OverlayTrigger
+                                        delayShow={Constants.OVERLAY_TIME_DELAY}
```
Comment (crspeller) [inline webapp/components/change_url_modal.jsx]: Unrelated?

**Comment by crspeller** [comment]
@asaadmahmoodspin Please remove unrelated changes from this PR. We accept unrelated changes on your other PRs only because its just style/CSS changes.

**Comment by asaadmahmoodspin** [comment]
Hey @crspeller I've removed those changes, have a look.
